File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if (len(s) != len(t)):
            return False;
        for i in s:
            if (t.count(i) >= 1):
                t = t[:t.index(i)] + t[t.index(i)+1:]
        if (len(t)>0):
            return False
        return True

File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        HashMap<Integer,Integer> count = new HashMap<Integer,Integer>();
        for (int i : nums){
            count.put(i,count.getOrDefault(i,0)+1);
        }
        for (int i : count.keySet()){
            if (count.get(i) > nums.length/2){
                return i;
            }
        }
        return 0;
    }
}